apps/meteor/app/livechat/server/business-hour/BusinessHourManager.ts

```diff
@@ -1,14 +1,17 @@
+import type { ILivechatBusinessHour, IBusinessHourTimezone } from '@rocket.chat/core-typings';
 import { LivechatBusinessHourTypes } from '@rocket.chat/core-typings';
-import type { ILivechatBusinessHour } from '@rocket.chat/core-typings';
 import type { AgendaCronJobs } from '@rocket.chat/cron';
-import { LivechatDepartment, Users } from '@rocket.chat/models';
+import { LivechatBusinessHours, LivechatDepartment, Users } from '@rocket.chat/models';
 import moment from 'moment';
 
 import { closeBusinessHour } from '../../../../ee/app/livechat-enterprise/server/business-hour/Helper';
 import { callbacks } from '../../../../lib/callbacks';
 import { settings } from '../../../settings/server';
 import type { IBusinessHourBehavior, IBusinessHourType } from './AbstractBusinessHour';
 
+const EVERY_MIDNIGHT_CRON_EXPRESSION = '0 0 * * *';
+const DAYLIGHT_CRON_JOB_NAME = 'livechat-business-hour-daylight-saving-time-verifier';
```

Comment:
```suggestion
const CRON_EVERY_MIDNIGHT_EXPRESSION = '0 0 * * *';
const CRON_DAYLIGHT_JOB_NAME = 'livechat-business-hour-daylight-saving-time-verifier';
```
This helps when I know what the resource is about that I'm searching, and can quickly scope down the autocomplete list by starting with that name.